### ISSUE TITLE
Webp

### ISSUE THREAD

**Issue by ThrowerWay** [Konloch/bytecode-viewer#479]
Hello, noticed that this project for a reason I do not yet know includes a native library webp-imageio.dll, this hasn't been updated since september so I am wondering if its perhaps vulnerable to CVE-2023-4863 ?

Could you update it if its vulnerable or just remove it (since why does a decompiler need webp decoding support?, and plus its the only native library in the jar as far as I can tell besides darklaf)

**Comment by Konloch** [comment]
We definitely could drop WebP support, it looks like the latest library is now https://github.com/gotson/NightMonkeys and requires JDK 21 support. If there are alternative libraries we could continue to support it, but I agree the easiest option is just drop it for now.

**Comment by ThexXTURBOXx** [comment]
I added WebP support because someone asked me in private for it and I also needed it for some of my projects (added in #329).
Indeed, if there is a better way for WebP support, we should migrate.

**Comment by ThexXTURBOXx** [comment]
I have opened a PR which fixes this issue by migrating to another project shipping new versions of libwebp.
I decided to do it this way because I think the fear around specific issues like CVE-2023-4863 is unjustified for the major part.
Just as an example: Apktool only recently fixed arbitrary write exploits in https://github.com/iBotPeaches/Apktool/commit/d348c43b24a9de350ff6e5bd610545a10c1fc712 and https://github.com/iBotPeaches/Apktool/commit/93e7d6bdbf366180bf402591b0feab72b2f568e3 and https://github.com/iBotPeaches/Apktool/commit/ea391dcd79ffe630ff67a895f8e2a4416225764a
Nobody knows about these issues - because the clientee of Apktool is different. Usually, people who use Apktool know about the risks and check apk files beforehand and so "nothing bad can happen".
By the way, BCV depends on Smali which depends on jcommander 1.64 which depends on libraries which are resolved through insecure HTTP channels: https://devhub.checkmarx.com/cve-details/Cx8fd408ac-dd80/?utm_source=jetbrains&utm_medium=referral&utm_campaign=idea
I think, issues like this are "okay" (and also to be expected) within professional applications like BCV.

Let me reiterate: I do **not** say that there is no issue with older versions of libwebp. I just think that the media did a great job of overexagerating the issue.
